### DISCUSSION TITLE
Join us for the Universe Keynote!  🎉

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#38695] [Announcements]
Join GitHub CEO, **Thomas Dohmke**, for the 2022 opening keynote live from the Yerba Buena Stage. He and other GitHub leaders will discuss the transforming nature of software development and how GitHub’s complete platform can be harnessed to empower all developers and enterprises. We’ve got some exciting announcements planned—you won’t want to miss it.

[Register Now](https://register.githubuniverse.com/) and [Join Live](http://watch.githubuniverse.com/)! 🎉 

![GitHub Universe 2022](https://user-images.githubusercontent.com/4041275/200900398-4f5f7df9-3608-4ce4-be7b-997d8ad19097.png)

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
